src/agenda.rs

```diff
@@ -471,6 +471,15 @@ pub fn lang<'a>() -> Box<dyn Action> {
                             exclude_labels: vec![],
                         }),
                     },
+                    QueryMap {
+                        name: "waiting_on_lang_team",
+                        kind: QueryKind::List,
+                        query: Box::new(github::Query {
+                            filters: vec![("state", "open")],
+                            include_labels: vec!["S-waiting-on-team", "T-lang"],
```

Comment:
This probably should be replicated for the next block (it currently only pulls from rust-lang/lang-team).

Reply:
@Mark-Simulacrum Fixed by moving it to the next block (which also covers lang-team AFAICT).